### PR TITLE
update DB::DumpSupportInfo to log whether jemalloc is supported or not

### PR DESCRIPTION
**Pull request by zaidoon1**

### CODE REVIEW

**Review by ajkr — APPROVED** [review]
Thanks!

File: db/db_impl/db_impl.cc [hunk 1]
```diff
@@ -71,6 +71,7 @@
 #include "options/cf_options.h"
 #include "options/options_helper.h"
 #include "options/options_parser.h"
+#include "port/jemalloc_helper.h"
```
Comment (jaykorean) [inline db/db_impl/db_impl.cc:74]: Should we put this `#include` also inside macro `#ifdef ROCKSDB_JEMALLOC`?

Comment (zaidoon1) [reply db/db_impl/db_impl.cc:74]: I was debating it but not sure what the style is for this repo? I can make the change

Comment (zaidoon1) [reply db/db_impl/db_impl.cc:75]: updated

**Pull request closed without merging**